Ginger/GingerCoreNET/Logger/WebReportGenerator.cs

```diff
@@ -129,61 +130,48 @@ public void DeleteFoldersData(string clientAppFolderPath)
 
         private void PopulateMissingFields(LiteDbRunSet liteDbRunSet, string clientAppPath)
         {
-            string imageFolderPath = Path.Combine(clientAppPath, "assets", "screenshots");
-            List<string> runSetEnv = new List<string>();
-            int totalRunners = liteDbRunSet.RunnersColl.Count;
-            int totalPassed = 0;
-            if (liteDbRunSet.RunStatus == eRunStatus.Automated.ToString())
-            {
-                totalPassed = liteDbRunSet.RunnersColl[0].BusinessFlowsColl.Where(bf => bf.RunStatus == eRunStatus.Passed.ToString()).Count();
-                liteDbRunSet.RunnersColl[0].RunStatus = liteDbRunSet.RunnersColl[0].BusinessFlowsColl[0].RunStatus;
-            }
-            else
+            //select template 
+            HTMLReportConfiguration _HTMLReportConfig = WorkSpace.Instance.SolutionRepository.GetAllRepositoryItems<HTMLReportConfiguration>().Where(x => (x.IsDefault == true)).FirstOrDefault();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4549121)